packages/demo/usecase-demo/src/components/Alert.tsx

```diff
@@ -16,18 +16,24 @@ export function Alert({
   return (
     <div
       className={cn(
-        'flex items-center gap-x-5 rounded-md border p-5 font-medium',
+        'flex items-center gap-x-5 rounded-lg border p-5 font-medium',
         variant === 'success'
           ? 'border-green-300 bg-success text-success-foreground'
           : variant === 'info'
             ? 'border-blue-300 bg-blue-50 text-blue-400'
-            : 'border-orange-300 bg-danger text-danger-foreground',
+            : 'border-[#9A1D1D] bg-[#FBCFCF] text-[#9A1D1D]',
         fullWidth ? 'w-full' : '',
         className
       )}
     >
       <div>
-        {variant === 'success' ? <CheckCircle size="20" /> : <Info size="20" />}
+        {variant === 'success' ? (
+          <CheckCircle size="24" />
+        ) : variant === 'error' ? (
+          <AlertOctagon size="24" />
+        ) : (
+          <Info size="24" />
+        )}
       </div>
```

Comment:
Unnecessary wrapping `<div>`?

Probably my fault, but if we can make it better in this PR, let's do it :)